aten/src/TH/generic/THTensorConv.cpp

```diff
@@ -1252,8 +1246,11 @@ void THTensor_(conv2Dmul)(THTensor *r_, scalar_t beta, scalar_t alpha, THTensor
 
   nelem = THTensor_(nElement)(r_);
   THTensor_(resize2d)(r_, nOutputRows, nOutputCols);
-  if (nelem == 0 || beta == 0 || nelem != THTensor_(nElement)(r_))
-    THTensor_(zero)(r_);
+  if (nelem == 0 || beta == 0 || nelem != THTensor_(nElement)(r_)) {
+    at::Tensor r_wrap = THTensor_wrap(r_);
+    r_wrap.zero_();
```

Comment:
This incurs an extra dispatch. Last time I measured, dispatch overhead was ~200ns. It *probably* doesn't matter too much

Reply:
Also, we're repeating
```
at::Tensor r_wrap = THTensor_wrap(r_);
r_wrap.zero_();
```
a lot. Maybe it's better to write a `THTensor_fill(TensorImpl*)` helper function?

As an alternative, this could just be a one-liner: `THTensor_wrap(r_).zero_();`.